lib/features/mailbox/domain/extensions/presentation_mailbox_extension.dart

```diff
@@ -29,4 +29,27 @@ extension PresentationMailboxExtension on PresentationMailbox {
        return imagePaths.icFolderMailbox;
      }
   }
+
+   String getChildOfTeamMailboxesIcon(ImagePaths imagePaths) {
+    switch(name!.name.toLowerCase()) {
+      case 'inbox':
+        return imagePaths.icMailboxInbox;
+      case 'drafts':
+        return imagePaths.icMailboxDrafts;
+      case 'archive':
+        return imagePaths.icMailboxArchived;
+      case 'sent':
+        return imagePaths.icMailboxSent;
+      case 'trash':
+        return imagePaths.icMailboxTrash;
+      case 'spam':
+        return imagePaths.icMailboxSpam;
+      case 'templates':
+        return imagePaths.icMailboxTemplate;
+      case 'all_mail':
+        return imagePaths.icMailboxAllMail;
```

Comment:
dont need this case, it is not in any spec. With `all mail`, icon will be normal directory